contrib/ci/drone-static-upload.sh

```diff
@@ -77,6 +80,10 @@ $mkdirs
 put $archive $upload_to
 SFTP
 
+if [ -e "$base-debug-symbols.tar.xz" ] ; then
+    sftp -i ssh_key -b - -o StrictHostKeyChecking=off drone@oxen.rocks <<<"put $base-debug-symbols.tar.xz $upload_to"
+fi
+
```

Comment:
Would be slightly cleaner (and marginally faster) to put this in a variable and stick it into the previous SFTP invocation, something like:
```
put_debug=
if [ -e "$base-debug-symbols.tar.xz" ] ; then
    put_debug="put $base-debug-symbols.tar.xz"
fi
```
...
```
sftp -i ssh_key -b - -o StrictHostKeyChecking=off drone@oxen.rocks <<SFTP
$mkdirs
put $archive $upload_to
$put_debug
SFTP
```